web/src/app/core/services/application.service.ts

```diff
@@ -3,23 +3,34 @@ import { AngularFirestore } from '@angular/fire/firestore';
 
 // Rxjs operators
 import { Observable } from 'rxjs';
+import { map, switchMap } from 'rxjs/operators';
 
 // Application model and services
-import { StatsModel } from '@shared/models/stats.model';
+import { IStats, StatsModel } from '@shared/models/index.model';
+import { ActivityService } from './activity.service';
 
 @Injectable({
   providedIn: 'root',
 })
 export class ApplicationService {
 
-  constructor(private afs: AngularFirestore) { }
+  constructor(
+    private afs: AngularFirestore,
+    private activityService: ActivityService
+
```

Comment:
We can remove the extra blank line in constructor

Reply:
Yep 👍 . If you refresh it should be gone 😄